lms/services/jwt_oauth2_token.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime, timedelta
+from functools import lru_cache
+from typing import Optional
+
+from lms.models.jwt_oauth2_token import JWTOAuth2Token
+
+
+class JWTOAuth2TokenService:
+    """Save and retrieve JWTOAuth2Tokens from the DB."""
+
+    EXPIRATION_LEEWAY = 60
+    """Amount of seconds to subtract to the expiration to give some time to account for request's delays."""
+
+    def __init__(self, db):
+        self._db = db
+
+    def save(
```

Comment:
```suggestion
    def save_token(
```
We might want to avoid very generic names for grepping purposes.